tasks/cron-task

```diff
@@ -1,3 +1,2 @@
-
 # run participate every minute
-*  *  *  *  *   bash ~/app/scripts/validationAndPreparation.sh
\ No newline at end of file
+*  *  *  *  *  . /root/project_env.sh;/app/scripts/validation_controller.sh >> /var/log/cron.log 2>&1
```

Comment:
Curious, how does it ensure we are not already running?

Reply:
Just saw your other comment. Get it now. Thanks!

Reply:
take a look into the validation_controller: It counts the number of processes running. If the cron job has started other processes the number is higher and otherwise, the number of running processes is higher. This is super hacky, but maybe you know a better solution